src/contracts/pods/EigenPod.sol

```diff
@@ -428,6 +427,38 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         _sendETH(recipient, amountWei);
     }
 
+
+    /*******************************************************************************
+                    EXTERNAL FUNCTIONS CALLABLE BY PERMISSIONED SERVICES
+    *******************************************************************************/
+
+    /// @notice Called by the EigenPodManager to fulfill a partial withdrawal proof request
+    function fulfillPartialWithdrawalProofRequest(
+        IEigenPodManager.Journal calldata journal,
+        uint64 feeGwei,
+        address feeRecipient
+    ) external onlyEigenPodManager onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_WITHDRAWAL) {
```

Comment:
can we create another pause index for zk pws?

Reply:
added